views/pages/invite.njk

```diff
@@ -4,12 +4,21 @@
     <h1>{{ __('invite.title') }}</h1>
 
     <div class="w-4/5 block border border-gray-500 border-solid p-5">
-        <p>{{ __('invite.give_code') }}</p>
+        <p class="font-semi-bold">{{ __('invite.give_code') }}</p>
+        <p>{{ __('invite.give_code_details') }}</p>
         <p class="font-bold text-6xl">{{ applicationCode }}</p>
```

Comment:
Can we move the code to the top of this div box? It's the most important piece of information/data on this page, and follows on from the new page heading.

Reply:
done